homework5.py

```diff
@@ -0,0 +1,95 @@
+"""
+Homework 5.
+
+Download a bunch  of files using:
+    - aiohttp, asyncio
+    - threads
+"""
+
+import asyncio
+import aiohttp
+
+import threading
+
+from time import time
+
+import urllib.request
```

Comment:
[requests](http://docs.python-requests.org) library is more convenient for using